### DISCUSSION TITLE
Chakra 3 UI components not displaying correctly in Next.js

### DISCUSSION THREAD

**Discussion by gekanchiko** [chakra-ui/chakra-ui#9394] [Q&A] [answered]
I am migrating from Chakra v2 to v3 in Next.js project (App Router) and faced with the issues with paddings for almost all basic components. There are no paddings in Button, Input, etc. Moreover, there is no way to set these paddings via recipes as they simply not applied. The only way to do so is to put padding prop directly to Button component that doesn't seem to be a good approach.

This is how button looks in an empty Next.js app:

<img width="568" alt="Screenshot 2024-12-28 at 23 30 01" src="https://github.com/user-attachments/assets/776f7a3c-a61d-40d3-b616-7f9466a0afdf" />

For example in Vite everything looking good, so the issue specifically in Next.js app.

**Comment by jessejuwe** [comment]
I have been experiencing the same issue as well in my Next.js 15 app.

**Comment by Pj-develop** [comment]
Same issue, sol is downgrade it

**Comment by segunadebayo** [comment, marked as answer]
Please remove the global css that comes with Next.js to resolve this.

Any custom css defined in the project takes more precedence over Chakra's styles.
